src/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -67,8 +67,15 @@ See the LICENSE file in the project root for more information.
   </PropertyGroup>
 
   <ItemGroup>
-    <DefaultFrameworkAssemblies Include="$(IlcPath)\sdk\*.dll" />
-    <DefaultFrameworkAssemblies Include="$(IlcPath)\framework\*.dll" />
+    <PrivateSdkAssemblies Include="$(IlcPath)\sdk\*.dll" />
```

Comment:
We can prevent this happening again by adding error checking to the beginning of the `_ComputeAssembliesToCompileToNative` target in Microsoft.NETCore.Native.Publish.targets.
```
<Error Condition="'@(PrivateSdkAssemblies)' == ''" Text="The PrivateSdkAssemblies ItemGroup is required for _ComputeAssembliesToCompileToNative" />
<Error Condition="'@(FrameworkAssemblies)' == ''" Text="The FrameworkAssemblies ItemGroup is required for _ComputeAssembliesToCompileToNative" />
```

Reply:
Awesome! Added.